### PR TITLE
feat: add Hero-section

### PR DESCRIPTION
**Pull request by Konovalova-Kateryna**

### CODE REVIEW

File: src/components/Hero/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import "./hero.scss";
+
+const Hero = () => {
+  return (
+    <section className="hero__container">
```
Comment (diana-tuz) [inline src/components/Hero/Hero.jsx:5]: the first class must named "hero" 
hero__container - it's name for the hero's child

File: src/components/Hero/hero.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
+@import '../../styles/variables';
+
+.hero__container{
```
Comment (diana-tuz) [inline src/components/Hero/hero.scss:3]: rename classes and use classical scss structure:
.hero{
  &__title{
    ...styles for hero__title
  }
 &__tumb{
...styles for hero__thumb
  }
}

**Review by diana-tuz — CHANGES_REQUESTED** [review]
need some changes to styles

File: src/components/Hero/hero.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    color: $white;
+    text-align: center;
+    padding-top: 180px;
+    padding-bottom: 180px;
+    padding-left: 16px;
+    padding-right:16px;
+    margin-left: auto;
+    margin-right: auto;    
+    @media screen and (min-width:($max-width-mobile)) {
+        width: $max-width-mobile;
```
Comment (diana-tuz) [inline src/components/Hero/hero.scss:35]: why did you change the width? Why for mobile you use width:100% and width at px for other versions?

File: src/components/Hero/hero.scss [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.hero__tumb{
+    width: 100%;
+    color: $white;
+    text-align: center;
+    padding-top: 180px;
+    padding-bottom: 180px;
+    padding-left: 16px;
+    padding-right:16px;
```
Comment (diana-tuz) [inline src/components/Hero/hero.scss:31]: padding: 180px 16px

it's the shortest form for it

File: src/components/Hero/hero.scss [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    margin-left: auto;
+    margin-right: auto;
```
Comment (diana-tuz) [inline src/components/Hero/hero.scss:33]: margin: 0 auto;

File: src/components/Hero/hero.scss [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    text-align: center;
+    padding-top: 180px;
+    padding-bottom: 180px;
+    padding-left: 16px;
+    padding-right:16px;
+    margin-left: auto;
+    margin-right: auto;    
+    @media screen and (min-width:($max-width-mobile)) {
+        width: $max-width-mobile;     
+    }
```
Comment (diana-tuz) [inline src/components/Hero/hero.scss:36]: change all elements where you use padding and margin and use the shortest form

https://developer.mozilla.org/en-US/docs/Web/CSS/margin

**Review by diana-tuz — CHANGES_REQUESTED** [review]

File: src/components/Hero/hero.scss (hunk 3 above)
Comment (Konovalova-Kateryna) [reply src/components/Hero/hero.scss]: up to the width of the mobile - responsive layout, after - adaptive

File: src/components/Hero/Hero.jsx (hunk 1 above)
Comment (Konovalova-Kateryna) [reply src/components/Hero/Hero.jsx]: done

File: src/components/Hero/hero.scss (hunk 2 above)
Comment (Konovalova-Kateryna) [reply src/components/Hero/hero.scss]: done

**Review by diana-tuz — CHANGES_REQUESTED** [review]
grid

**Review by diana-tuz — APPROVED** [review]

**Comment by diana-tuz** [comment]
awesome! but you didn't add Hero to App and I can't watch it at preview(
